lib/google_maps.ex

```diff
@@ -851,6 +851,197 @@ defmodule GoogleMaps do
+      application can be found in the [Place Photos](https://developers.google.com/places/web-service/photos) documentation.
+      A photo object is described as:
+
+      * `photo_reference` — a string used to identify the photo when you perform a Photo request.
+
+      * `height` — the maximum height of the image.
+
+      * `width` — the maximum width of the image.
+
+      * `html_attributions[]` — contains any required attributions. This field will always be present, but may be empty.
+
+    * `place_id` - a textual identifier that uniquely identifies a place. To retrieve information about the place,
+      pass this identifier in the placeId field of a Places API request. For more information about place IDs, 
+      see the [place ID overview](https://developers.google.com/places/web-service/place-id).
+
+    * `scope` - Indicates the scope of the place_id. The possible values are:
+
+      * `APP`: The place ID is recognised by your application only. This is because your application added the place,
+        and the place has not yet passed the moderation process.
+
+      * `GOOGLE`: The place ID is available to other applications and on Google Maps.
+
+    * `alt_ids` — An array of zero, one or more alternative place IDs for the place, 
+      with a scope related to each alternative ID. Note: This array may be empty or not present. 
+      If present, it contains the following fields:
+
+      * `place_id` — The most likely reason for a place to have an alternative place ID is if your application
+        adds a place and receives an application-scoped place ID, then later receives a Google-scoped place ID
+        after passing the moderation process.
+
+      * `scope` — The scope of an alternative place ID will always be APP, indicating that the alternative
+        place ID is recognised by your application only.
+
+    * `price_level` — The price level of the place, on a scale of 0 to 4. The exact amount indicated by a
+      specific value will vary from region to region. Price levels are interpreted as follows:
+
+      * `0` — Free
+
+      * `1` — Inexpensive
+
+      * `2` — Moderate
+
+      * `3` — Expensive
+
+      * `4` — Very Expensive
+
+    * `rating` contains the place's rating, from 1.0 to 5.0, based on aggregated user reviews
+
+    * `types` contains an array of feature types describing the given result. 
+      See the [list of supported types](https://developers.google.com/places/web-service/supported_types#table2).
+
+    * `vicinity` contains a feature name of a nearby location. Often this feature refers to a street or
+      neighborhood within the given results.
+
+    * `permanently_closed` is a boolean flag indicating whether the place has permanently shut down (value true).
+      If the place is not permanently closed, the flag is absent from the response.
+
+  ## Examples
+
+    # Search for museums 500 meters around the White house 
```

Comment:
I may be wrong and need to generate the docs to check, but I believe that you do need a full 4 space indentation after `##Examples` for them to show up as code block.

Reply:
I'll generate the docs locally and fix this.